docs/installation/instructions.rst

```diff
@@ -1,10 +1,19 @@
 Installation Instructions
 =========================
 
+Supported Platforms
+-------------------
+
+Pulp has been demonstrated to work on Ubuntu, Debian, Fedora, CentOS, and Mac OSX, but it should
+work on any operating system that can provide a Python 3.5+ runtime environment and the supporting
```

Comment:
Can we reverse the order of these statements. The primary claim is the runtime environment so I'm hoping that will be first.